providers/keycloak/tests/unit/keycloak/auth_manager/test_keycloak_auth_manager.py

```diff
@@ -59,3 +88,228 @@ def test_serialize_user(self, auth_manager):
     def test_get_url_login(self, auth_manager):
         result = auth_manager.get_url_login()
         assert result == f"{AUTH_MANAGER_FASTAPI_APP_PREFIX}/login"
+
+    @pytest.mark.parametrize(
+        "function, method, details, permission_type, permission_resource",
+        [
+            [
+                "is_authorized_configuration",
+                "GET",
+                ConfigurationDetails(section="test"),
+                "Configuration#GET",
+                "Configuration:test#GET",
+            ],
+            ["is_authorized_configuration", "GET", None, "Configuration#GET", None],
+            [
+                "is_authorized_configuration",
+                "PUT",
+                ConfigurationDetails(section="test"),
+                "Configuration#PUT",
+                "Configuration:test#PUT",
+            ],
+            [
+                "is_authorized_connection",
+                "DELETE",
+                ConnectionDetails(conn_id="test"),
+                "Connection#DELETE",
+                "Connection:test#DELETE",
+            ],
+            ["is_authorized_connection", "GET", None, "Connection#GET", None],
+            ["is_authorized_backfill", "POST", BackfillDetails(id=1), "Backfill#POST", "Backfill:1#POST"],
+            ["is_authorized_backfill", "GET", None, "Backfill#GET", None],
+            ["is_authorized_asset", "GET", AssetDetails(id="test"), "Asset#GET", "Asset:test#GET"],
+            ["is_authorized_asset", "GET", None, "Asset#GET", None],
+            [
+                "is_authorized_asset_alias",
+                "GET",
+                AssetAliasDetails(id="test"),
+                "AssetAlias#GET",
+                "AssetAlias:test#GET",
+            ],
+            ["is_authorized_asset_alias", "GET", None, "AssetAlias#GET", None],
+            [
+                "is_authorized_variable",
+                "PUT",
+                VariableDetails(key="test"),
+                "Variable#PUT",
+                "Variable:test#PUT",
+            ],
+            ["is_authorized_variable", "GET", None, "Variable#GET", None],
+            ["is_authorized_pool", "POST", PoolDetails(name="test"), "Pool#POST", "Pool:test#POST"],
+            ["is_authorized_pool", "GET", None, "Pool#GET", None],
+        ],
+    )
+    @pytest.mark.parametrize(
```

Comment:
Double stacked parametrize, neat! Does this basically doe the cross product between the two sets of args?

Reply:
Cool!